src/Processor/UrlFromDoiProcessor.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/*
+ * This file is part of the BibTex Parser.
+ *
+ * (c) Renan de Lima Barbosa <renandelima@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace RenanBr\BibTexParser\Processor;
+
+use RenanBr\BibTexParser\Exception\ProcessorException;
+
+class UrlFromDoiProcessor
+{
+    use TagSearchTrait;
+
+    private $urlDoiPrefix;
+
+    public function __construct($urlDoiPrefix = 'https://doi.org')
```

Comment:
That's very nice, I liked the `$urlDoiPrefix` argument

Could you make some changes ?

- Place default value in a class constant, as `https://doi.org/%s`
- Change default argument value to `null`, then pick it from the constant when necessary
- Change its name (and property) to `$urlDoiFormat`, then use `sprintf()` in `__invoke()`

Reply:
Done !